server/sonar-server/src/main/java/org/sonar/server/app/TomcatStartupLogs.java

```diff
@@ -26,12 +26,12 @@
 import org.sonar.api.utils.log.Logger;
 import org.sonar.process.Props;
 
-class StartupLogs {
+class TomcatStartupLogs {
 
   private final Logger log;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "log" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)